api/routes.py

```diff
@@ -18,8 +22,9 @@ def create_redflag():
             "status": 400
             }), 400
     data = request.get_json()
-    if 'createdBy' not in data or 'comment' not in data or 'type' not in data or 'location' not in data or 'status' not in data:
```

Comment:
this is the update I think so you can look through the merge conflicts here in this text editor and remove the old  conflicting lines of code directly and then merge after

Reply:
Sorry for responding a bit late. But i have never resolved conflicts onlne